prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -142,9 +142,9 @@ class LookupTableEndpointUtilities(val environment: Environment) {
      * @throws TableNotFoundException if the table and/or version is not found
      * @throws IOException if there is a server or API error
      */
-    fun createTable(tableName: String, tableData: List<Map<String, String>>):
+    fun createTable(tableName: String, tableData: List<Map<String, String>>, force: Boolean):
```

Comment:
Change the name of the argument called force to be more descriptive and update the kdoc

Reply:
Done.  Changed to forceTableToCreate.